tests/plugins/balloonpanel/balloonpanel.js

```diff
@@ -149,16 +149,47 @@
 			assert.isTrue( doc.getActive().equals( panelElement ), 'Panel element is focused' );
 
 			// Giving focus element explicitly.
-			panel.attach( strong, closeElement );
+			panel.attach( strong, { focusElement: closeElement } );
 			assert.isTrue( doc.getActive().equals( closeElement ), 'Close element is focused' );
 
 			// Blur focused element, so the body will be focused.
 			closeElement.$.blur();
 
 			// Ensure that focus is not changed when giving false.
-			panel.attach( strong, false );
+			panel.attach( strong, { focusElement: false } );
 			assert.isTrue( doc.getActive().equals( doc.getBody() ), 'Focus remains on body element' );
 		},
+		'test panel show option': function() {
+			var panel = new CKEDITOR.ui.balloonPanel( bender.editor, {
+				title: 'Test panel #3'
+			} ),
+				strong = bender.editor.document.findOne( 'strong' ),
+				showEvent = false;
+
+			panel.addShowListener( function() {
```

Comment:
Instead of manually write handling for checking whether or not given method was called, you should use `sinon.stub()`. Familiarize yourself with sinon lib in general, it's a great testing lib that has a lot of good features.

Here's an example how it can look like: https://gist.github.com/mlewand/4e3fef172a69593bb5d5e844b1b4bea0